twingate/internal/provider/resource/resource.go

```diff
@@ -66,6 +66,7 @@ func Resource() *schema.Resource { //nolint:funlen
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Optional:    true,
 				Description: "List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API",
+				Deprecated:  "Configure `access` block instead. This attribute will be removed in the next version of the provider.",
```

Comment:
Let's change this to: "The group_ids argument is now deprecated, and the new access block argument should be used instead. The group_ids argument will be removed in a future version of the provider."

Reply:
updated